build.gradle

```diff
@@ -8,12 +8,12 @@ description = 'A Gradle plugin that detects the OS name and architecture, provid
 group = 'com.google.gradle'
 
 // The major and minor versions are aligned with the Maven plugin's.
-version = '1.4.0'
+version = '1.4.1'
```

Comment:
The version should be `1.4.1-SNAPSHOT`. I should've made the change right after the `1.4.0` release.